spec/directions/bake_example_spec.rb

```diff
@@ -99,6 +101,58 @@
     end
   end
 
+  context 'when there is a table' do
+    let(:table) do
+      <<~HTML
+        <div class="os-table">
+          <table class="some-class" id="tId">
+            <thead>
+              <tr>
+                <th>A title</th>
+              </tr>
+              <tr>
+                <th>Another heading cell</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>One lonely cell</td>
+              </tr>
+            </tbody>
+          </table>
+          <div class="os-caption-container">
+            <span class="os-title-label">Table </span>
+            <span class="os-number">S</span>
+            <span class="os-divider"> </span>
+            <span class="os-caption">
+              <span data-type="title">Secret Title</span>
+            </span>
+          </div>
+        </div>
+      HTML
+    end
+
+    it 'doesn\'t affect the baked table' do
+      described_class.v1(example: example, number: 4, title_tag: 'title-tag-name')
+      expect(example).to match_normalized_html(
+        <<~HTML
+          <div data-type="example" id="example-test">
+            <title-tag-name class="os-title">
+              <span class="os-title-label">Example </span>
+              <span class="os-number">4</span>
+              <span class="os-divider"> </span>
+            </title-tag-name>
+            <div class="body">
+              <h4 data-type="title">example title becomes h4</h4>
+              <p>content</p>
+              #{table}
+            </div>
+          </div>
+        HTML
+      )
+    end
+  end
```

Comment:
I'm not quite understand how this part works is it checking if the table is placed exactly in the example? :sweat_smile:

Reply:
there's a line in `BakeExample` that converts a `data-type=["title"]` span to an h4. But this shouldn't impact other elements contained within the example. It was impacting my tables, so I changed the line & added this spec to make sure 😄